packages/react-error-overlay/src/utils/mapper.js

```diff
@@ -34,7 +34,11 @@ async function map(
   });
   await settle(
     files.map(async fileName => {
-      const fileSource = await fetch(fileName).then(r => r.text());
+      const fetchUrl = fileName.startsWith('webpack-internal:')
+        ? `/getInternalSource?fileName=${encodeURIComponent(fileName)}`
```

Comment:
Can we export this URL from a package? That way if we change it we don't need to update two places.
See https://github.com/facebook/create-react-app/blob/next/packages/react-dev-utils/launchEditorEndpoint.js

Reply:
Also, we should probably make this more in line with the other url, i.e. `/__get-internal-source`.

Reply:
> /__get-internal-source.

👍 

> Can we export this URL from a package? 

I'm a bit worried about importing that module in react-error-overlay. It looks like react-dev-utils might be a new dependency.